ras/diag_encl.py

```diff
@@ -51,12 +53,14 @@ def test_diag_encl(self):
         diag_path = '/var/log/ppc64-diag/diag_disk/'
         if not os.path.isdir(diag_path):
             self.fail('diag_disk path does not exists.')
-        if not self.run_cmd_out("diag_encl -h | grep -Eai 'disk health'").strip():
+        if not self.run_cmd_out("diag_encl -h |"
```

Comment:
@PraveenPenguin Since this line is being looked at, can it be changed to python ?
parsing in python intead of grep.